src/NuGet.DependencyResolver.Core/Providers/IDependencyProvider.cs

```diff
@@ -12,5 +12,7 @@ public interface IDependencyProvider
         bool SupportsType(string libraryType);
 
         Library GetDescription(LibraryRange libraryRange, NuGetFramework targetFramework);
+
+        IEnumerable<string> GetAttemptedPaths(NuGetFramework targetFramework);
```

Comment:
Talked to @davidfowl , will try to avoid this change and put attempted paths to somewhere else later